app/Views/DemoPages/CustomGestures/CustomMapGestureView.swift

```diff
@@ -32,6 +47,8 @@ class CustomMapGestureView: UIView, IMapGestureView {
 		pinchGR.delegate = self
 		self.addGestureRecognizer(pinchGR)
 		self.pinchGestureRecognizer = pinchGR
+
+		self.addGestureRecognizer(self.rotationGestureRecognizer!)
```

Comment:
Не пиши лишний раз !, не смущай людей.
В одну строку можно так.
```suggestion
		self.rotationGestureRecognizer.map(self.addGestureRecognizer)
```

Reply:
СДелал